web/src/components/upload-indicator/upload-indicator.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import { IndeterminateBar, ProgressBar } from '@epam/loveship';
+import styles from './upload-indicator.module.scss';
+import { UploadProgressTracker } from 'connectors/documents-page-control-connector/use-fetch-with-tracking-of-upload-progress';
+
+export const UploadIndicator: FC<{
```

Comment:
Is it possible to show more visible background for the progress bar?
<img width="91" alt="image" src="https://github.com/epam/badgerdoc/assets/13063442/b66ff948-8b19-4b40-9e68-a673990ca160">

Reply:
addressed